usage/admin/classes/domain/SushiService.php

```diff
@@ -432,6 +432,18 @@ private function jsonTransfer($reportLayout, $serviceProvider)
       $reportErrorCodes = array(2000, 2010, 2020, 3000, 3010, 3020, 3030);
       $errorsOrFatalCount = 0;
       foreach($json->Report_Header->Exceptions as $exception) {
+
+        // 1011 means the report was queued, in this case, try again after 1 second
+        // after three retries, exit
+        if ($exception->Code == 1011) {
+          if ($retry == 3) {
+            $this->logStatus("Report was queued for processing, but was not processable after 3 retries.");
+            $this->saveLogAndExit($reportLayout);
+          }
+          sleep(1);
```

Comment:
I think we should back out this commit. Using JSTOR as the only example of a vendor that returns error 1011 that we are aware of, the wait time varies considerably, depending on the number of reports you request and the date range you specify. Even with a very long sleep interval, I don’t get any reports more often than I do. The user can always manually rerun the import from the console.